### ISSUE TITLE
Using built-in Help will spam you with error logs

### ISSUE THREAD

**Issue by septN** [godotengine/godot#77178]
### Godot version

4.1.dev (da21cb7)

### System information

Windows 11

### Issue description

There's a nasty bug going on with the latest master branch. The built-in help will spit out hundreds of errors in Editor Log. And if you keep navigating through, it will eventually crash the IDE. 

![image](https://github.com/godotengine/godot/assets/13846022/15a41721-d099-46fa-820e-08abff3587de)

![image](https://github.com/godotengine/godot/assets/13846022/f441c83e-40fe-4bff-9c16-7234ea2c8033)

![image](https://github.com/godotengine/godot/assets/13846022/cdc00a57-b815-418d-8ef1-7e52a9605e97)

### Steps to reproduce

1. Open a new project
2. Create a new scene, save it.
3. Create a new script, save it.
4. Open the Script Editor, and perform Ctrl+Click on the class name (ie. Node2D)

### Minimal reproduction project

N/A

**Comment by hakro** [comment]
Yeah, the documentation is pretty much broken as well. Can't scroll down or do anything.

![2023-05-18_22-20](https://github.com/godotengine/godot/assets/3624853/d0abc03c-824b-45dc-a59a-399f8ec6587d)

**Comment by Calinou** [comment]
This is likely caused by the recent threading changes (https://github.com/godotengine/godot/pull/75901 and/or https://github.com/godotengine/godot/pull/77000).
